### PR TITLE
Add influxdb to arkade

### PR DESCRIPTION
**Pull request by Waterdrips**
Adding influxdb to arkade, with --persistence flag to request a pvc

This has been tested both on and off.

Extra args can be added in the future as we see fit.

Signed-off-by: Alistair Hey <alistair@heyal.co.uk>

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
As discussed on live stream! and https://github.com/alexellis/arkade/issues/381

## How Has This Been Tested?

locally using 

```sh
arkade install influxdb

# and

arkade install influxdb --persistence
```
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [ ] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

- [ ] I have tested this on arm, or have added code to prevent deployment
https://hub.docker.com/layers/influxdb/library/influxdb/1.8/images/sha256-e762a7cc5f2fb705673b012870b836c6efd396040772d8131845c98a40acb914?context=explore
armv7, amd64 and arm64 multi-arch image

### CODE REVIEW

File: cmd/apps/influxdb_app.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+		Example:      "arkade install influxdb --persistence",
+		SilenceUsage: true,
+	}
+
+	influxdbApp.Flags().StringP("namespace", "n", "default", "The namespace to install chartmuseum (default: default")
+	influxdbApp.Flags().Bool("update-repo", true, "Update the helm repo")
+	influxdbApp.Flags().Bool("persistence", false, "Enable persistence for influxdb (default: false)")
+
+	influxdbApp.RunE = func(command *cobra.Command, args []string) error {
+		kubeConfigPath, _ := command.Flags().GetString("kubeconfig")
```
Comment (alexellis) [inline cmd/apps/influxdb_app.go:30]: Do we need `kubeconfig` here? I thought we moved it up higher for each command.

File: cmd/apps/influxdb_app.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+		println(influxdbInstallMsg)
+		return nil
+	}
+
+	return influxdbApp
+}
+
+const InfluxdbInfoMsg = `
+# Get started with influxdb here:
```
Comment (alexellis) [inline cmd/apps/influxdb_app.go:65]: Can you send a follow-up PR to wrap to 80 chars please?

Possibly also make it more succinct like a port-forward with CLI access.

The approach of getting a pod name isn't needed for deployments, you can just specify its name

**Review by alexellis — APPROVED** [review]
Approved, but please can you send a follow-up PR for the comments?

File: cmd/apps/influxdb_app.go (hunk 1 above)
Comment (ah-f3) [reply cmd/apps/influxdb_app.go:30]: we need to grab it from here, but its defined at top level up

**Comment by alexellis** [comment]
Do we need updates in the docs given the move away from `GO_REPLACE`? https://docs.openfaas.com/cli/templates/#go-golang-http-of-watchdog-template

**Pull request merged**